datafusion/functions/src/unicode/rpad.rs

```diff
@@ -46,8 +48,11 @@ impl RPadFunc {
             signature: Signature::one_of(
                 vec![
                     Exact(vec![Utf8, Int64]),
```

Comment:
As noted in https://github.com/apache/datafusion/pull/11924/files::

// Planner attempts coercion to the target type starting with the most preferred candidate.
// For example, given input `(Utf8View, Utf8)`, it first tries coercing to `(Utf8View, Utf8View)`.
// If that fails, it proceeds to `(Utf8, Utf8)`.

Thus, the ordering of the signature should likely be changed to reflect that.

Reply:
Sure